sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -864,9 +864,9 @@ class MapboxMap internal constructor(
    */
   fun removeFeatureState(
     sourceId: String,
-    sourceLayerId: String?,
+    sourceLayerId: String? = null,
     featureId: String,
-    stateKey: String?
+    stateKey: String? = null
```

Comment:
Are you sure it works as expected? if having 3 parameters of type `String?` where 2 of them have default value.

Reply:
yes, users can used named parameters in this case.